experimental/benchmarks/acmeair/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:14.04
+MAINTAINER Gareth Ellis <gareth.ellis@uk.ibm.com>
+RUN apt-get update && apt-get install git openjdk-7-jdk mongodb zip unzip numactl bc wget -y
+RUN mkdir /node 
+RUN date
+WORKDIR /node
+RUN mkdir Jmeter node_undertest node_baseline
+RUN git clone http://github.com/gareth-ellis/benchmarking 
+RUN bash /node/benchmarking/experimental/benchmarks/acmeair/setupJmeter.sh
+RUN numactl --hardware
```

Comment:
Similar to the above statement, can we do this in the same layer as `apt-get install` so we can purge the dep? I'm not familiar with numactl.